Documentation/Signoffs/Mario Kart Rev 2 - Signoff - RRSA System.md

```diff
@@ -0,0 +1,86 @@
+
+### Constraints
+
+**Measurement accuracy:** it is imperative that the device record the altitude of the rider with a high degree of precision. If the simulated resistance is to be within 10 % of the real-world resistance as previously constrained, the combined inaccuracy of the pressure sensor, the speed sensor, and the resistance system must be below 10 %. Let the error allowance for the determination of the energy expended during exercise be 1 %. For a bike and rider of 91 kg, an altitude gain of 4.67 m results in an energy expenditure of 1 kCal since E = mgh. Accordingly, the required precision to achieve +/- 1 % accuracy at a granularity of 1 kCal is +/- 4.67 cm. The speed must also be sampled at a rate which allows reasonably fast response time. Since the use of single-magnet speed sensing is already proven as common practice for bike computers, and all potential energy calculations can be performed with only the measured altitude over time, the main constraint for speed sensing is one of durability. The reed switch used must nominally tolerate at least 1000 miles of operation. 
+
+**Video recording:** the resolution is to be 720p, and the frame rate is to be at least 30 fps. 
+
+**Battery life:** the device must be able to continuously operate for at least 3 hours, and must provide the user with a visual indication of battery charge status. 
+
+**Recording capacity:** the device must be able to capture 3 hours of continuous footage and measurements. Additionally, the storage media must possess the bandwidth to record the video as it is being taken. 
+
+**Usability:** the device must provide an easily used interface, which is also unobtrusive and does not distract the rider. 
+
+**Durability:** the device must be shielded from outside weather conditions, and all component boards must be attached firmly enough to prevent them from being knocked around during riding.
+
+### Schematic:
+
+![RRSA Schematic](https://user-images.githubusercontent.com/118228609/203230366-a49b449b-943c-4f00-ad5b-cdeb976bec66.png)
+Figure 1: a schematic for the assembly. Not pictured: the camera, which is to be attached to the camera port on the Raspberry Pi Zero 2W.
+
+![PIC03042](https://user-images.githubusercontent.com/118228609/203231527-3cd93c0c-69ab-4919-86b1-4a5de69bc6ae.JPG)
+Figure 2: a drawing illustrating the placement of modules within the project box.
+
+![PIC03041](https://user-images.githubusercontent.com/118228609/203231351-a6e881cc-bc34-441a-97ff-cbec0ddabef7.JPG)
+Figure 3: a drawing illustrating the front and back panels. The first item (from left to right) on the front panel is a panel-mount USB-C socket. The second is the power switch, followed by the start/stop switch. The hole in the back panel is for the camera.
+
+### Analysis:
```

Comment:
For the sensing appropriateness and battery life, more numeric analysis is needed.

Reply:
Battery life analysis completed.